app/assets/stylesheets/members/index.css.scss

```diff
@@ -12,16 +15,31 @@ $parallax-window-height: 40em;
 $parallax-background-height: $parallax-window-height;
 
 
-.parallax-window {
-  max-height: $parallax-window-height;
-  height: $parallax-window-height;
-  overflow: hidden;
-  position: relative;
-  text-align: center;
-  width: 100%;
-  margin-top: $gutter;
-  margin-bottom: $gutter;
+@include breakpoint($desktop) {
+  .parallax-window {
+    max-height: $parallax-window-height;
```

Comment:
Properties should be ordered height, margin-bottom, margin-top, max-height, overflow, position, text-align, width